go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.15.1
 	github.com/bnb-chain/zkbnb-smt v0.0.3-0.20221227064653-7422bfd51aa0
 	github.com/consensys/gnark v0.7.0
-	github.com/consensys/gnark-crypto v0.7.0
+	github.com/consensys/gnark-crypto v0.10.0
```

Comment:
We should keep the version of gnark-crypto unchanged, Because we are actually using our own modified version of gnark-crypto, see the "replace" field.